site/docs/numbers/porting/csrLookupTool.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: csrLookupTool
 title: CSR Lookup Tool
-slug: /numbers/guides/csrLookupTool
+slug: /numbers/guides/porting/csrLookupTool
 description: How to lookup CSR records using the Bandwidth API
```

Comment:
Can we rename instances of "CSR Lookup Tool" to just "CSR Lookup"? Please do this in all other files in this PR as well. I want to avoid using the term "Tool" here.

Reply:
Could you change the filename as well to just csrLookup instead of csrLookupTool?  I think there are two instances of the filename in this PR so need to change both.

Reply:
oops, missed that sorry, thanks for catching this!